### ISSUE TITLE
Not pushing to WearOS

### ISSUE THREAD

**Issue by vOidward** [aneonex/BitcoinChecker#106]
Notifications have stopped pushing to wear OS sometime recently. They appear in the list of notifications, but they do not show or vibrate when a new one should be pushed to the smartwatch.

**Comment by mangblekok** [comment]
Yes I can confirm this too, my smartwatch can't pick up the notification.
